<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditview</class>
 <widget class="QWidget" name="creditview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: grey;
}</string>
  </property>
  <widget class="QLabel" name="SpanLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PaymentLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;	
}</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="OverallPayment">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;	
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MonthlyPayment">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;	
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Overpayment">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>330</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;	
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="MeasurementBox">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>116</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: darkGray;
	color: black;
}</string>
   </property>
   <item>
    <property name="text">
     <string>лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>месяцев</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinSum">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>51</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>999999999999999983222784.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCalculate">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	background-color: black;
	border-radius: 2px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #9799fc);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Radio1">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {	
	background-color: darkGray;
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Аннуитентные</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="SumLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="OverpaymentLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>330</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;	
}</string>
   </property>
   <property name="text">
    <string>Переплата</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PercentLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TypeLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип ежемесячных платежей</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinPercent">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="OverallPaymentLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;	
}</string>
   </property>
   <property name="text">
    <string>Общая сумма платежа</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinSpan">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="Radio2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>270</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {	
	background-color: darkGray;
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
  </widget>
  <widget class="QLabel" name="Image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
